src/tests/class_tests/openms/source/IdentificationData_test.cpp

```diff
@@ -50,13 +50,17 @@ START_TEST(IdentificationData, "$Id$")
 using namespace OpenMS;
 using namespace std;
 
-IdentificationData* ptr = 0;
-IdentificationData* null = 0;
+IdentificationData* ptr = nullptr;
+IdentificationData* null = nullptr;
 START_SECTION((IdentificationData()))
   ptr = new IdentificationData();
   TEST_NOT_EQUAL(ptr, null);
 END_SECTION
 
+START_SECTION((~IdentificationData()))
+  delete ptr;
```

Comment:
this will give a warning, since there is nothing to test here.
Either omit the section or add `NOT_TESTABLE`

Reply:
is omitting a good idea? I though ASan could detect e.g., missing virtual destructors